util/MultiplayerCommon.h

```diff
@@ -75,6 +68,12 @@ struct FO_COMMON_API GalaxySetupData {
     std::map<std::string, std::string>
                         m_game_rules;
     std::string         m_game_uid;
+
+    /** HACK! This must be set to the encoding empire's id when serializing a
+      * Universe, so that only the relevant parts of the Universe are
+      * serialized.  The use of this global variable is done just so I don't
```

Comment:
update this comment... unlike the main encoding empire global, this is not a global...

Reply:
Fixed.